IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -319,16 +319,18 @@ - (BOOL)matchesWithRealm:(nullable NSString *)realm
     {
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->